notes/NewSiteArchitecture.md

```diff
@@ -0,0 +1,174 @@
+# New GraphQL.org Site Architecture:
+
+## Index
+
+*Goal:* This is the landing page and is our opportunity to quickly capture attention and explain what GraphQL is and why you should care.
+
+*Timeframe:* Launch Mon, Sept 12th
+
+This page is effectively a marketing page for GraphQL and should be the visual, scrollable version of the "Introducing GraphQL" conference talks and should be rich with visual metaphor and illustration and take advantage of whitespace to make individual salient points.
+
+Above the fold, this page should succinctly explain what GraphQL is and illustrate with a simple (editable) query/response example. Before scrolling you should understand the following:
+
+* GraphQL is an query language for APIs.
+* GraphQL is sent by client applications, such as an iOS app.
+* GraphQL is evaluated by a web service and often returned as JSON.
+* GraphQL typically replaces REST.
```

Comment:
> If you have half your API in REST and half of it in GraphQL, that seems like a bad idea.

We have something like this in our production app right now, because we didn't have time to migrate the whole thing, and it's actually not bad at all! We ended up making the critical parts of our app's UI faster, and we can migrate other parts as we want to.

> Definitely if you are starting a new project, don't plan on it being half in REST and half in GraphQL.

Agreed. I'm referring specifically to people who have REST already. Basically, I think current GraphQL materials over-emphasize building new apps, and don't have the right messaging for companies who already have a lot of APIs and are looking for a solution to make them better over time.